ddpui/api/airbyte_api.py

```diff
@@ -43,13 +43,15 @@
 )
 from ddpui.utils.singletaskprogress import SingleTaskProgress
 
+from fastapi import Request  # Added for type hinting
+
 airbyte_router = Router()
 logger = CustomLogger("airbyte")
 
 
 @airbyte_router.get("/source_definitions", auth=auth.CustomAuthMiddleware())
 @has_permission(["can_view_sources"])
-def get_airbyte_source_definitions(request):
+def get_airbyte_source_definitions(request: Request) -> List[Dict]:
```

Comment:
Hi @AyushDineshRathi for api endpoints we want to type annotate using Django's response model. 

If you look at the file `airbyte_api.py` carefully, this has already been done for some apis.
```
@airbyte_router.post(
    "/v1/connections/",
    auth=auth.CustomAuthMiddleware(),
    response=AirbyteConnectionCreateResponse,
)
```

You will first need to understand , what the api returns and create response models for them